test/unit/configmap_spec.js

```diff
@@ -2,108 +2,105 @@ const should = require('should') // eslint-disable-line
+        it('has host', function () {
+            ffYML.should.have.property('host')
+            ffYML.host.should.equal('0.0.0.0')
         })
-        it('has domain',  function () {
-            ff_yml.should.have.property('domain')
-            ff_yml.domain.should.equal('example.com')
+        it('has domain', function () {
+            ffYML.should.have.property('domain')
+            ffYML.domain.should.equal('example.com')
         })
         it('has port', function () {
-            ff_yml.should.have.property('port')
+            ffYML.should.have.property('port')
         })
         describe('has db', function () {
             it('has db', function () {
-                ff_yml.should.have.property('db')
+                ffYML.should.have.property('db')
             })
             it('has db.type', function () {
-                ff_yml.db.should.have.property('type')
-                ff_yml.db.type.should.equal('postgres')
+                ffYML.db.should.have.property('type')
+                ffYML.db.type.should.equal('postgres')
             })
         })
         describe('has driver', function () {
             it('has driver', function () {
-                ff_yml.should.have.property('driver')
+                ffYML.should.have.property('driver')
             })
             it('has driver.type', function () {
-                ff_yml.driver.should.have.property('type')
-                ff_yml.driver.type.should.equal('kubernetes')
+                ffYML.driver.should.have.property('type')
+                ffYML.driver.type.should.equal('kubernetes')
             })
         })
         describe('has logging', function () {
             it('has logging', function () {
-                ff_yml.should.have.property('logging')
+                ffYML.should.have.property('logging')
             })
             it('default', function () {
-                ff_yml.logging.should.have.property('level')
-                ff_yml.logging.level.should.equal('info')
+                ffYML.logging.should.have.property('level')
+                ffYML.logging.level.should.equal('info')
             })
             it('http', function () {
-                ff_yml.logging.should.have.property('http')
-                ff_yml.logging.http.should.equal('info')
+                ffYML.logging.should.have.property('http')
+                ffYML.logging.http.should.equal('info')
             })
         })
         describe('has broker', function () {
-            let mosquitto_conf
-            before( function () {
-                const brokerCM = configMaps.filter(cm => cm.metadata.name === 'flowforge-broker-config')[0]
-                mosquitto_conf = brokerCM.data['mosquitto.conf']
+            // let mosquittoConf
+            before(function () {
+                // const brokerCM = configMaps.filter(cm => cm.metadata.name === 'flowforge-broker-config')[0]
+                // mosquittoConf = brokerCM.data['mosquitto.conf']
```

Comment:
Why were these removed? I assume because the test that makes use of it on lines 83-85 don't exist yet?

Reply:
> Why were these removed?

Flagged as unused variables and I couldn't see anything that hinted at them being needed... but I did go for commenting out rather than removing so they can be easily restored.